docs/src/language_servers.md

```diff
@@ -542,6 +542,34 @@ There are multiple options:
     }
     ```
 
+### Steep
+
+1. Add the `steep` gem (see [github:soutaro/steep](https://github.com/soutaro/steep)) into your Gemfile and install it
+
+    ```sh
+    bundle install
+    ```
+
+2. Binstub steep executable
+
+    ```sh
+    steep binstub
+    ```
+
+3. Open `Preferences > Package Settings > LSP > Settings` and add the `"steep"` client configuration to the `"clients"`:
+
+    ```jsonc
+    {
+        "clients": {
+            "steep": {
+                "enabled": true,
+                "command": ["bin/steep", "langserver"],
```

Comment:
Is this supposed to be a relative path? Because it is here but since this is a global server configuration, it's not likely to work.